sql/core/src/test/scala/org/apache/spark/sql/sources/v2/TestInMemoryTableCatalog.scala

```diff
@@ -252,6 +252,32 @@ class InMemoryTable(
       withData(messages.map(_.asInstanceOf[BufferedRows]))
     }
   }
+
+  private def splitAnd(filter: Filter): Seq[Filter] = {
```

Comment:
Please don't duplicate methods like this. I suggest moving the original method to a companion object if you want to use it elsewhere.

Reply:
Fixed also.